marker/renderers/html.py

```diff
@@ -24,9 +24,21 @@ class HTMLOutput(BaseModel):
 
 
 class HTMLRenderer(BaseRenderer):
-    page_blocks: list = [BlockTypes.Page]
-    paginate_output: bool = False
-    image_extraction_mode: Literal["lowres", "highres"] = "highres"
+    """
+    A renderer for HTML output.
+    """
+    page_blocks: Annotated[
+        List[BlockTypes],
+        "The block types to consider as pages.",
+    ] = [BlockTypes.Page]
```

Comment:
Default types shouldn't be lists

Reply:
I know this was here from before, but worth fixing - it's on the jsonrenderer too